src/components/MinimalDemoWithWeather.js

```diff
@@ -67,13 +82,15 @@ const GetWeatherToolkit = {
 };
 
 const MinimalAgent = {
-	id: 'tiny',
-	name: 'Conversation Bot',
-	toolkits: [GetWeatherToolkit],
+	id: 'tiny-with-weather',
+	name: 'Conversation Bot with weather',
+	toolkits: [GetWeatherToolkit.name],
```

Comment:
I wonder if here if the `GetWeatherToolkit` object is passed in rather than the name, and that toolkit is not registered, we could register it rather than having to use `useToolkit`?

Reply:
I was playing around with this a little bit. These are [some changes made to Big Sky Agents](https://github.com/Automattic/big-sky-agents/compare/try/register-toolkits-with-agent?expand=1) to also register the toolkits after an agent is registered.

The agent toolkit definition would look like:
```
toolkits: [GetWeatherToolkit],
```

and it would remove the need to call `useToolkit`

Reply:
That makes sense to me @kthai15, I would go ahead and make that a PR.

Reply:
@glendaviesnz opened one [here](https://github.com/Automattic/big-sky-agents/pull/41)